zkp/build.rs

```diff
@@ -0,0 +1,16 @@
+use std::{env, path::Path};
+
+use nimiq_compile_zkp_setup::verifying_keys_setup;
+use nimiq_zkp_primitives::NanoZKPError;
+
```

Comment:
I still think we can just put the function from `compile-zkp-setup` here.
To test this, maybe copy the function into the `build.rs` first and see whether it works.
Only remove the additional crate afterwards.